src/models/user-model.js

```diff
@@ -1,48 +1,48 @@
-import {v4 as uuidv4} from 'uuid';
-
-const users = [];
-
-module.exports = class User {
-    constructor(login, password, age) {
-        this.id = uuidv4();
-        this.login = login;
-        this.password = password;
-        this.age = age;
-        this.isDeleted = false;
-    }
-
-    save() {
-        users.push(this);
+import Sequelize from "sequelize";
+import database from "../config/database";
```

Comment:
please fix eslint errors, seems like your linter is not fully configured, as I get lots of eslint errors on my PC. Please check if you have eslint and prettier extensions installed
![image](https://user-images.githubusercontent.com/47612379/84073852-3d062900-a9da-11ea-9232-4b72a5040c0e.png)